prog/main.go

```diff
@@ -162,12 +201,14 @@ func logCensoredArgs() {
 
 func main() {
 	var (
-		flags         = flags{}
-		mode          string
-		debug         bool
-		weaveEnabled  bool
-		weaveHostname string
-		dryRun        bool
+		flags                            = flags{}
+		mode                             string
+		debug                            bool
+		weaveEnabled                     bool
+		weaveHostname                    string
+		dryRun                           bool
+		containerLabelFilterFlags        = containerLabelFiltersFlag{exclude: false}
```

Comment:
Minor: This is equivalent to `containerLabelFiltersFlag{}`. Although it may be clearer to make it explicit. Your call.